RestApiForGroup2/server.js

```diff
@@ -0,0 +1,199 @@
+const req = require('request');
+const pg = require('pg');
+const express = require('express');
+const router = express.Router();
+const path = require('path');
+const body_parser = require('body-parser');
+const  HttpStatus  =  require('http-status-codes');
+const hostname = '127.0.0.1';
+const port = 3000;
+
+var app = express();
+app.use(express.static(__dirname + '/public'));
+app.use(body_parser.json());
+app.use(body_parser.urlencoded({
+    extended: true
+}));
+
+var cn = {
+    host: 'localhost',
+    port: 5432,
+    database: 'postgres',
+    user: 'postgres',
+    password: '1234',
```

Comment:
Please put this into a config file and include an example configuration. See last years group: https://github.com/Rostlab/JS16_ProjectA/tree/master/cfg